chainerx_cc/chainerx/python/routines.cc

```diff
@@ -350,6 +350,23 @@ void InitChainerxManipulation(pybind11::module& m) {
           [](const ArrayBodyPtr& a, int8_t axes) { return MoveArrayBody(Transpose(Array{a}, {axes})); },
           py::arg("a"),
           py::arg("axes") = nullptr);
+    m.def("pad",
+          [](const ArrayBodyPtr& a, int8_t pad_width, const std::string& mode, int64_t constant_values) {
+              return MoveArrayBody(Pad(Array{a}, pad_width, mode, constant_values));
+          },
+          py::arg("a"),
```

Comment:
Let's use argument name "array" to align with `numpy.pad`.

Reply:
Done!